app/models/scheme.rb

```diff
@@ -15,6 +15,8 @@ class Scheme < ApplicationRecord
             format: { with: URI::MailTo::EMAIL_REGEXP },
             allow_blank: true
 
+  scope :within_14_months, (-> { where(start_date: [14.months.ago..Time.zone.now]) })
```

Comment:
What's special about the 14 month cut off? I'm guessing it's "a little bit more than a year". The way it's coded up it reads a bit of a magic number. If you're able to name the concept/reason behind it I think that would be better.

The figure is also used three times (here, and in `ReportController` and `CombinedReportController`). If, for some reason, this was shifted to 15 months instead of 14, would all of those places need to change? If so, it'd be nice to have it defined in one place – possibly as a constant on `Scheme`. If not, it would be good to name the concept for those places to distinguish them.

Reply:
Great spot, I deferred naming this concept whilst using it until I found something that worked, and then of course completely forgot about it!